source/rules/literals.rs

```diff
@@ -722,78 +640,6 @@ mod tests {
     };
 
     #[test]
-    fn case_null() {
-        let input  = Span::new(b"null");
```

Comment:
I guess it's better to ensure that special constants are correctly parsed ;-). So I prefer to move them.